src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -356,38 +369,38 @@ void func_80B85900(EnKaizoku* this) {
     }
 }
 
-s32 func_80B85A00(EnKaizoku* this, PlayState* play, s16 arg2) {
+s32 EnKaizoku_ReactToPlayer(EnKaizoku* this, PlayState* play, s16 arg2) {
     Player* player = GET_PLAYER(play);
-    s16 sp32 = ABS_ALT(this->picto.actor.wallYaw - this->picto.actor.shape.rot.y);
-    s16 sp30 = ABS_ALT(this->picto.actor.yawTowardsPlayer - this->picto.actor.shape.rot.y);
+    s16 angleToWall = ABS_ALT(this->picto.actor.wallYaw - this->picto.actor.shape.rot.y);
+    s16 angleToLink = ABS_ALT(this->picto.actor.yawTowardsPlayer - this->picto.actor.shape.rot.y);
```

Comment:
```suggestion
    s16 angleToPlayer = ABS_ALT(this->picto.actor.yawTowardsPlayer - this->picto.actor.shape.rot.y);
```

There are a few of things could you change them all?